navigation/dead_reckoning/sam_dead_reckoning/launch/sam_dr_launch.py

```diff
@@ -4,88 +4,71 @@
 from ament_index_python.packages import get_package_share_directory
 from launch_ros.actions import Node
 
+# TODO xml version of launch file
 
 def generate_launch_description():
-    # Example: using a yaml file for parameters
-    # param_config = os.path.join(
-    #     get_package_share_directory("sam_graph_slam_2"),
-    #     "config",
-    #     "pipeline_config.yaml"
-    # )
-
-    # TODO this is not the correct way to set the name
-    # I think we should talk about how launch files should be set up
-    robot_name = "sam0"
-    odom_topic = f"/{robot_name}/dr/odom"
-    gps_odom_topic = f"/{robot_name}/dr/gps_odom"
+    namespace = "sam0"
+    # These topics are specific to sam dr
+    depth_topic = f"{namespace}/dr/depth"
+    odom_topic = f"/{namespace}/dr/odom"
+    gps_odom_topic = f"/{namespace}/dr/gps_odom"
 
     print("Launching sam_dr_launch.py")
 
     return launch.LaunchDescription([
-        # Example: ExecuteProcess
-        # ExecuteProcess(
-        #     cmd=["ros2", "bag", "play", "/home/julian/bag_files/pipeline_sim"],
-        #     output="screen"
-        # ),
-        # TODO include the latlon to utm service
+        # TODO include the latlon to utm service ??
         Node(
             package="sam_dead_reckoning",
             executable="depth_node",
+            namespace=namespace,
             name="depth_node",
             output="screen",
             parameters=[{
-                "odom_frame":  f"{robot_name}/odom",
-                "base_frame": f"{robot_name}/base_link",
-                "pressure_frame": f"{robot_name}/pressure_link",
-                "pressure_topic": f"/{robot_name}/core/depth20_pressure",
-                "depth_topic": f"/{robot_name}/dr/depth",
+                "odom_frame":  f"{namespace}/odom",
+                "base_frame": f"{namespace}/base_link",
+                "pressure_frame": f"{namespace}/pressure_link",
+                "pressure_topic": f"/{namespace}/core/depth20_pressure",
+                "depth_topic": depth_topic,
                 "simulation": True
```

Comment:
Can you add a TODO to change this param to instead check a topic (it is not there yet, but will be there eventually.)
Instead of manually changing this, the sim can emit a topic that is missing in the real thing and we can reduce one more point of human failure 👍